### ISSUE TITLE
Missing Documentation

### ISSUE THREAD

**Issue by Pixdigit** [DexterInd/BrickPi_Python#65]
There is little to no documentation about this module.
At least a listing of all functions and classes with a brief explanation would be helpful.
Also way to little function are presented in the examples folder.

Greetings, Pixdigit.

**Comment by graykevinb** [comment]
I agree. I would also like to add to that, a list of function is nice, but also how to use the sensors would be very helpful. I did figure it out, but it could be easier.

**Comment by burf2000** [comment]
There is no documentation on install BrickPi if you're not using their image
